backend/Dockerfile

```diff
@@ -28,14 +28,20 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/app
 # --- Stage 2 ---
 # Docker will use the last stage of your Dockerfile as the actual
 # image. In this case, use a completely empty image
-FROM scratch
+#FROM scratch -- "x509: certificate signed by unknown authority" error when accessing firebase
+FROM alpine
```

Comment:
instead of using `alpine` you can add this line to add network certs and git:

```Dockerfile
RUN apk add git ca-certificates
```

Reply:
it seems like the scratch image doesn't have apk installed: https://stackoverflow.com/a/40211742/11223254 
`
FROM scratch
RUN apk add git ca-certificates
`
gave: 
`
Step 11/16 : RUN apk add git ca-certificates
 ---> Running in eca7ceea97fa
OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown
`

Reply:
ok jokes I forgot about that, there's technically a way to copy to certs over but it's very hacky. Let's leave it as `alpine` then!